src/main/java/no/sikt/nva/pubchannels/dataporten/DataportenJournal.java

```diff
@@ -0,0 +1,110 @@
+package no.sikt.nva.pubchannels.dataporten;
+
+import static java.util.Objects.isNull;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import no.sikt.nva.pubchannels.Immutable;
+import no.sikt.nva.pubchannels.handler.ScientificValue;
+import no.sikt.nva.pubchannels.handler.ThirdPartyJournal;
+
+/* default */ final class DataportenJournal implements Immutable, ThirdPartyJournal {
```

Comment:
default?

Reply:
Was recommended for package visible classes in a different repository to indicate it is intentional. Tried removing it, and no warning here. Removing it.